### DISCUSSION TITLE
Problems building on Windows

### DISCUSSION THREAD

**Discussion by bingxuan-li** [assimp/assimp#4602] [Q&A] [answered]
I was building assimp with Visual Studio 2022 on Windows and I got errors like this. 
What's wrong and how do I fix it? 
Thanks

![屏幕截图 2022-07-01 230734](https://user-images.githubusercontent.com/73156236/176921152-a2381ab0-e35c-4ef5-bf56-36e898027481.png)

**Comment by kimkulling** [comment, marked as answer]
This was caused by a different gtest which is in use. Make sure that the gtest installation in the assimp repo comes first in your include variable.
